contracts/dapis/DapiServer.sol

```diff
@@ -699,13 +699,31 @@ contract DapiServer is
         return (dataFeed.value, dataFeed.timestamp);
     }
 
-    /// @notice Reads the dAPI with name
+    /// @notice Reads the data feed value with ID
+    /// @param dataFeedId Data feed ID
+    /// @return value Data feed value
+    function readDataFeedValueWithId(bytes32 dataFeedId)
+        external
+        view
+        override
+        returns (int256 value)
```

Comment:
Shouldn't this be `int224`? If not, should we explicitly cast `dataFeed.value` to `int256`?

Reply:
I initially wanted the interface to look simple but thinking about it a bit more, you're right. One read function returning `int256` while the other returns `int224` implies they are returning different things and can be confusing in its own regard. I'll use `int224`, it doesn't change anything in practice anyway.